dingrogu-client/src/main/java/org/jboss/pnc/dingroguclient/DingroguBuildPushDTO.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2022 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.dingroguclient;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+import org.jboss.pnc.api.dto.Request;
+
+import java.util.List;
+
+@Value
+@Builder
+@Jacksonized
+public class DingroguBuildPushDTO {
```

Comment:
You will also need
```
String causewayUrl;    
String orchUrl;
```
You also want username in causeway push dto so I guess that one too? 
Or is this class just placeholder and will be replaced with DTO from dingrougu once my PR is ready?

Reply:
I believe this is just a placeholder